src/Data/Fin/Mod.agda

```diff
@@ -0,0 +1,107 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- ℤ module n
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Data.Fin.Mod where
+
+open import Function using (_$_; _∘_)
+open import Data.Bool using (true; false)
+open import Data.Nat as ℕ using (ℕ; s≤s)
+open import Data.Nat.Properties as ℕ
+  using (m≤n⇒m≤1+n; 1+n≰n; module ≤-Reasoning)
+open import Data.Fin as F hiding (suc; pred; _+_; _-_)
+open import Data.Fin.Properties
+open import Relation.Nullary using (Dec; yes; no; contradiction)
```

Comment:
There's a drift (which I am actively pursuing in #2055) towards narrowing this to the following:
```
open import Relation.Nullary.Decidable.Core using (Dec; yes; no)
open import Relation.Nullary.Negation.Core using (contradiction)
```